pontoon/base/static/js/translate.js

```diff
@@ -657,14 +656,18 @@ var Pontoon = (function (my) {
       $("#original").on("click", ".placeable", function (e) {
         e.stopPropagation();
         e.preventDefault();
+        $('#translation').focus();
```

Comment:
For some reason, i couldn't get selectionStart/selectionEnd properties without this line. However, it works fine by now. Thanks!